.github/actions/install-aiida-services/action.yml

```diff
@@ -0,0 +1,6 @@
+name: install-aiida-services
```

Comment:
it seems like these actions are not used in the main .yml file or am I mistaking?

Reply:
yes, you are right. I removed them.